### PR TITLE
Example titles

### PR DESCRIPTION
**Pull request by Malmz**

### CODE REVIEW

File: druid/examples/layout.rs [hunk 1]
```diff
@@ -53,7 +53,9 @@ fn build_app() -> impl Widget<u32> {
 }
 
 fn main() {
-    let window = WindowDesc::new(build_app);
+    let window = WindowDesc::new(build_app).title(
+        LocalizedString::new("layout-demo-window-title").with_placeholder("Very flexable".into()),
```
Comment (futurepaul) [inline druid/examples/layout.rs]: typo: `flexable`

File: druid/resources/i18n/en-US/builtin.ftl [hunk 2]
```diff
@@ -1,6 +1,10 @@
 -app-name = Druid
 
 hello-counter = Current value is { $count }
+either-demo-window-title = {$view ->
```
Comment (cmyr) [inline druid/resources/i18n/en-US/builtin.ftl]: fun :)

**Comment by Malmz** [comment]
No pressure, it's really fun contributing to a real project

**Comment by cmyr** [comment]
Thanks!

**Pull request merged**